src/__tests__/Call.test.ts

```diff
@@ -525,6 +540,45 @@ describe('Call class', () => {
         await expect(postFeedbackPromise).resolves.toBe(undefined);
       });
     });
+
+    describe('.sendMesssage', () => {
```

Comment:
Can we add negative test cases?

Reply:
I could only think of 1 negative case for this method. Take a look, and let me know if you have another case I should implement.

Reply:
Can you link the commit here? Sorry I should've written down what I was thinking in the first review. I'm thinking cases like the ones we're checking for here https://github.com/twilio/twilio-voice.js/blob/master/lib/twilio/call.ts#L904
- send empty params
- send missing properties etc

Reply:
commit to throw error if param is not `CallMessage`. https://github.com/twilio/twilio-voice-react-native/pull/272/commits/e1a7ea775cfa530d019741d106026dd230db2ab3 

Let me try adding those other 2 cases

Reply:
@charliesantos additional negative cases: https://github.com/twilio/twilio-voice-react-native/pull/272/commits/61036c0927a2393845084a51f0b606af27e56cf8